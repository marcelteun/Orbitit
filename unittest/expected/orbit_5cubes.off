OFF
#
# SimpleShape
#
# file generated with python script by Marcel Tunnissen
# Vertices Faces Edges
40 30 60
# Vertices
1 -1 1
1.618033988749895 0 -0.618033988749894
0.618033988749895 1.618033988749895 0
0 0.618033988749895 1.618033988749895
-0.618033988749895 -1.618033988749895 0
0 -0.618033988749895 -1.618033988749895
-1 1 -1
-1.618033988749895 0 0.618033988749894
-1 -1 1
1 -1 1
1 1 1
-1 1 1
-1 -1 -1
1 -1 -1
1 1 -1
-1 1 -1
-1.618033988749896 0.000000000000001 -0.618033988749894
-0.618033988749894 1.618033988749896 0.000000000000001
1 1 -1.000000000000001
-0.000000000000001 -0.618033988749895 -1.618033988749896
-1 -1 1.000000000000001
0.000000000000001 0.618033988749895 1.618033988749896
1.618033988749896 -0.000000000000001 0.618033988749894
0.618033988749894 -1.618033988749896 -0.000000000000001
0.000000000000001 -0.618033988749895 1.618033988749895
1.000000000000001 1 1
-0.618033988749895 1.618033988749895 0
-1.618033988749895 0 0.618033988749896
0.618033988749895 -1.618033988749895 0
1.618033988749895 0 -0.618033988749896
-0.000000000000001 0.618033988749895 -1.618033988749895
-1.000000000000001 -1 -1
-0.618033988749896 -1.618033988749894 -0.000000000000001
0.999999999999999 -1 -1
1.618033988749895 -0.000000000000001 0.618033988749895
0 -0.618033988749895 1.618033988749894
-1.618033988749895 0.000000000000001 -0.618033988749895
0 0.618033988749895 -1.618033988749894
0.618033988749896 1.618033988749894 0.000000000000001
-0.999999999999999 1 1
# Sides and colours
4  0 1 2 3  254 214 0
4  0 3 7 4  254 214 0
4  1 0 4 5  254 214 0
4  2 1 5 6  254 214 0
4  3 2 6 7  254 214 0
4  7 6 5 4  254 214 0
4  8 9 10 11  33 138 33
4  8 11 15 12  33 138 33
4  9 8 12 13  33 138 33
4  10 9 13 14  33 138 33
4  11 10 14 15  33 138 33
4  15 14 13 12  33 138 33
4  16 17 18 19  138 0 0
4  16 19 23 20  138 0 0
4  17 16 20 21  138 0 0
4  18 17 21 22  138 0 0
4  19 18 22 23  138 0 0
4  23 22 21 20  138 0 0
4  24 25 26 27  0 190 254
4  24 27 31 28  0 190 254
4  25 24 28 29  0 190 254
4  26 25 29 30  0 190 254
4  27 26 30 31  0 190 254
4  31 30 29 28  0 190 254
4  32 33 34 35  138 133 77
4  32 35 39 36  138 133 77
4  33 32 36 37  138 133 77
4  34 33 37 38  138 133 77
4  35 34 38 39  138 133 77
4  39 38 37 36  138 133 77
# END
