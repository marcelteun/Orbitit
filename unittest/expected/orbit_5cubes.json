{"class": "orbit_shape", "data": {"cols": [[0.996094, 0.839844, 0], [0.132812, 0.542969, 0.132812], [0.542969, 0, 0], [0, 0.746094, 0.996094], [0.542969, 0.523438, 0.304688]], "final_sym": {"class": "A5", "data": {"generator": {"o3axis": [1, 1, 1], "o5axis": [0, 1.6180339887, 1]}}}, "fs": [[0, 1, 2, 3], [0, 3, 7, 4], [1, 0, 4, 5], [2, 1, 5, 6], [3, 2, 6, 7], [7, 6, 5, 4]], "name": "orbit_5cubes", "stab_sym": {"class": "A4", "data": {"generator": {"o2axis0": [1, 0, 0], "o2axis1": [0, 1, 0]}}}, "vs": [[1, 1, 1], [-1, 1, 1], [-1, -1, 1], [1, -1, 1], [1, 1, -1], [-1, 1, -1], [-1, -1, -1], [1, -1, -1]]}}